classes/ImageManager.php

```diff
@@ -354,39 +354,33 @@ public static function imagecopyresampled(
     }
 
     /**
-     * Check if file is a real image.
-     *
-     * @param string $filename File path to check
-     * @param string $fileMimeType File known mime type (generally from $_FILES)
-     * @param array $mimeTypeList Allowed MIME types
+     * @param string $filename
      *
-     * @return bool
+     * @return string|bool
      */
-    public static function isRealImage($filename, $fileMimeType = null, $mimeTypeList = null)
+    public static function getMimeType(string $filename)
     {
-        // Detect mime content type
         $mimeType = false;
-        if (!$mimeTypeList) {
-            $mimeTypeList = array('image/gif', 'image/jpg', 'image/jpeg', 'image/pjpeg', 'image/png', 'image/x-png');
-        }
-
-        // Try 4 different methods to determine the mime type
+        // Try with GD
         if (function_exists('getimagesize')) {
             $imageInfo = @getimagesize($filename);
-
             if ($imageInfo) {
                 $mimeType = $imageInfo['mime'];
-            } else {
-                $fileMimeType = false;
             }
-        } elseif (function_exists('finfo_open')) {
+        }
+        // Try with FileInfo
+        if (!$mimeType && function_exists('finfo_open')) {
             $const = defined('FILEINFO_MIME_TYPE') ? FILEINFO_MIME_TYPE : FILEINFO_MIME;
```

Comment:
You can use directly the const, it was for compatiblity with <5.3
```suggestion
            $const = FILEINFO_MIME_TYPE;
```

Reply:
:+1: